pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -51,8 +62,8 @@ var corefileTemplate = template.Must(template.New("Corefile").Funcs(template.Fun
         {{$.LogLevel}}
     }
     bufsize 512
-    cache 900 {
-        denial 9984 30
+    cache {{ $.PositiveTTL }} {
+        denial 9984 {{ $.NegativeTTL }}
```

Comment:
```suggestion
    cache  {
        success 9984 {{ $.PositiveTTL }}
        denial 9984 {{ $.NegativeTTL }}
```

Reply:
It's not necessary since we don't do any additional configuration of caching. We ship with this today:
```
cache 900 {
  denial 9984 30
}
```

Reply:
Imo it would be clearer to spell out each of the success and denial values separately, so we don't get bug reports or questions asking us why it doesn't show up, but in the end I guess it's a bit of a nit.
```suggestion
    cache {
      success 9984 900
      denial 9984 30
      }
```